### DISCUSSION TITLE
Saving Scenes Not Working As Intended

### DISCUSSION THREAD

**Discussion by jasoon-chen** [empriselab/RCareWorld#126] [Help - Programming] [answered]
I'm trying to load the scene for the dressing environment however none of them works.

I tried saving the scene and then loading the scene into the environment. I know that there is an example in `test_scene.py` that offers sample code on how to save a scene and load one, but it does not work as intended on my ended. If I change the provided example code such as by changing the `SaveScene()` function call it does not make the json file. The provided example code for this is in `test_scene.py`
```
env.SaveScene("test_scene.json")
env.ClearScene()
env.Pend()

env.LoadSceneAsync("test_scene.json")
env.Pend()
env.close()
```
However, if I change the `test_scene.json` in `SaveScene()` and `LoadSceneAsync()` to `some_random_file_name.json` I do not see that json file under `/Player_Data/StreamingAssets/SceneData`. This is the code that I used:
```
env.SaveScene("some_random_file_name.json")
env.ClearScene()
env.Pend()

env.LoadSceneAsync("some_random_file_name.json")
env.Pend()
env.close()
```
I read the documentation and it mentions that [SaveScene() ](https://rcareworld.readthedocs.io/en/latest/_modules/pyrcareworld/envs/base_env.html#RCareWorld.SaveScene) would save default saving to the `StreamingAssets` folder."

Is this producible on your end?

**Comment by YoruCathy** [comment]
HI @jasoon-chen which player are you using?

**Reply by jasoon-chen** [reply]
In the `test_scene.json` file I was using the player provided which is `player_path = os.path.join(executable_path, "../executable/Player/Player.x86_64")`. However, I also tested in this the Dressing environment and I was not able to find the `.json` file under `StreamingAssets` Folder.

**Reply by DanielTakeshi** [reply]
Hello @YoruCathy 
Has there been any follow-up on this?

**Reply by YoruCathy** [reply]
Hi @DanielTakeshi we'll look into this soon.

**Reply by YoruCathy** [reply]
I documented an issue for this in https://github.com/empriselab/RCareWorld/issues/145

**Reply by YoruCathy** [reply, marked as answer]
This issue is resolved!
